pbf/views.py

```diff
@@ -1139,16 +1139,27 @@ def remove_element_permanent(request, player_id, element):
     player.save()
 
     compute_card_thresholds(player)
-    return with_log_trigger(render(request, 'player.html', {'player': player}))
+    return with_log_trigger(render(request, 'player.html', {'player': player, 'button_or_a': button_or_a(request)}))
 
 def tab(request, game_id, player_id):
     game = get_object_or_404(Game, pk=game_id)
     player = get_object_or_404(GamePlayer, pk=player_id)
     compute_card_thresholds(player)
-    return render(request, 'tabs.html', {'game': game, 'player': player})
+    return render(request, 'tabs.html', {'game': game, 'player': player, 'button_or_a': button_or_a(request)})
 
 def game_logs(request, game_id):
     game = get_object_or_404(Game, pk=game_id)
     logs = reversed(game.gamelog_set.order_by('-date').all()[:30])
     return render(request, 'logs.html', {'game': game, 'logs': logs})
 
+def button_or_a(request):
+    return request.COOKIES.get('button_or_a', 'button')
```

Comment:
This is a security issue, though difficult to exploit. Any consequences would likely be self-inflicted. I apologise for not noticing it earlier. I will fix it now.